src/deps.ts

```diff
@@ -10,11 +10,63 @@ export { copy, readAll, writeAll } from 'jsr:@std/io@0.225.2'
 export * as sqlite from 'jsr:@db/sqlite@0.12.0'
 export { type Database as SQLiteDB } from 'jsr:@db/sqlite@0.12.0'
 export * as esbuild from 'npm:esbuild@0.25.6'
+export { z } from 'npm:zod@4.0.5'
 export { default as tweetnacl } from 'npm:tweetnacl@1.0.3'
 export { base58btc } from 'npm:multiformats@11.0.2/bases/base58'
 export { type Multibase } from 'npm:multiformats@11.0.2'
 export { default as blake } from 'npm:@multiformats/blake2@1.0.13'
 export { CID } from 'npm:multiformats@11.0.2/cid'
+
+// Serve command dependencies (pinned to Group Income versions)
+export { default as sbp } from 'npm:@sbp/sbp@2.4.1'
+export { Server as Hapi } from 'npm:@hapi/hapi@20.1.2'
+export type { Request, ResponseToolkit, Server as ServerType } from 'npm:@hapi/hapi@20.1.2'
+export { default as Boom } from 'npm:@hapi/boom@9.1.0'
+export { default as Joi } from 'npm:@hapi/joi@17.1.1'
+export { default as Inert } from 'npm:@hapi/inert@6.0.3'
+export { default as chalk } from 'npm:chalk@4.1.0'
+export { default as pino } from 'npm:pino@8.19.0'
+export { default as LRU } from 'npm:lru-cache@7.14.0'
+export * as Three from 'npm:three@0.151.3'
+export { default as BetterSqlite3 } from 'npm:better-sqlite3@11.9.1'
+export { default as WebSocket, WebSocketServer } from 'npm:ws@8.5.0'
+export { default as Vue } from 'npm:vue@2.7.16'
+export { mixin as clickaway } from 'npm:vue-clickaway@2.2.2'
+export { default as Vuex } from 'npm:vuex@3.6.0'
+export { default as VueRouter } from 'npm:vue-router@3.6.5'
+export { default as pug } from 'npm:pug@3.0.2'
+export { default as dompurify } from 'npm:dompurify@2.2.7'
+export { default as should } from 'npm:should@13.2.3'
+export { v4 as uuid } from 'npm:uuid@9.0.0'
+export { cloneDeep, omit } from 'npm:turtledash@1.0.3'
+
+export { default as Bottleneck } from 'npm:bottleneck@2.19.5'
+export { default as scrypt } from 'npm:scrypt-async@2.0.1'
+
+// RFC8188 encryption dependencies
+export { aes128gcm } from 'npm:@apeleghq/rfc8188@1.0.7/encodings'
+export { default as rfc8188Encrypt } from 'npm:@apeleghq/rfc8188@1.0.7/encrypt'
+
+// Chelonia dependencies
+export { blake32Hash, createCID, maybeParseCID, multicodes, strToB64, getSubscriptionId, parseCID } from 'npm:@chelonia/lib@1.2.0/functions'
+export { checkKey, parsePrefixableKey, prefixHandlers } from 'npm:@chelonia/lib@1.2.0/db'
+export { SPMessage } from 'npm:@chelonia/lib@1.2.0/SPMessage'
+export { SERVER } from 'npm:@chelonia/lib@1.2.0/presets'
+export { ChelErrorGenerator } from 'npm:@chelonia/lib@1.2.0'
```

Comment:
If you're using path imports, this should be imported from `/errors`. Also, note that the version to import now is `1.2.2`.

Reply:
Yep, noted.
Updated @chelonia/lib to 1.2.2